src/main/java/org/assertj/core/error/ElementsShouldSatisfy.java

```diff
@@ -29,4 +33,12 @@ private ElementsShouldSatisfy(Object actual, Object notSatisfies, String asserti
           actual, notSatisfies, assertionErrorDetails);
   }
 
+  private ElementsShouldSatisfy(Object actual) {
+    super("%n" +
+                    "Expecting any element of:%n" +
+                    "  <%s>%n" +
+                    "to satisfy given requirements, but did not.",
+            actual);
```

Comment:
Small nitpick: This formatting does not seem to be correct.

Otherwise the pull request looks great. 👍